x-pack/plugins/session_view/public/components/DetailPanelProcessTab/index.test.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { AppContextTestRender, createAppRootMockRenderer } from '../../test';
+import { DetailPanelProcess, DetailPanelProcessLeader } from '../../types';
+import { DetailPanelProcessTab } from './index';
+
+const getLeaderDetail = (leader: string): DetailPanelProcessLeader => ({
+  id: `${leader}-id`,
+  name: `${leader}-name`,
+  start: new Date('2022-02-24'),
+  entryMetaType: 'sshd',
+  userName: `${leader}-jack`,
+  interactive: true,
+  pid: 1234,
+  entryMetaSourceIp: '10.132.0.50',
+  executable: '/usr/bin/bash',
+});
+
+const TEST_PROCESS_DETAIL: DetailPanelProcess = {
+  id: 'process-id',
+  start: new Date('2022-02-22'),
+  end: new Date('2022-02-23'),
+  exit_code: 137,
+  user: 'process-jack',
+  args: ['vi', 'test.txt'],
+  executable: [
+    ['test-executable-cmd', '(fork)'],
+    ['test-executable-cmd', '(exec)'],
+    ['test-executable-cmd', '(end)'],
+  ],
+  pid: 1233,
+  entryLeader: getLeaderDetail('entryLeader'),
+  sessionLeader: getLeaderDetail('sessionLeader'),
+  groupLeader: getLeaderDetail('groupLeader'),
+  parent: getLeaderDetail('parent'),
+};
+
+describe('DetailPanelProcessTab component', () => {
+  let render: () => ReturnType<AppContextTestRender['render']>;
+  let renderResult: ReturnType<typeof render>;
+  let mockedContext: AppContextTestRender;
+
+  beforeEach(() => {
+    mockedContext = createAppRootMockRenderer();
+  });
+
+  describe('When DetailPanelProcessTab is mounted', () => {
+    it('renders DetailPanelProcessTab correctly', async () => {
```

Comment:
Thanks! Same thing here about any error or loading states?

Reply:
All the data should be loaded already so there won't be more fetches when user access the detail panel. I will reach out to our design team on the error state. Thanks for pointing it out! :)